Chapter_9/eks/README.md

```diff
@@ -0,0 +1,8 @@
+## Inferencing using Elastic Kubernetes Service
+
+This first section of this chapter describes EKS-based approach. We introduce how to create inference endpoints for TensorFlow (TF) and PyTorch models and deploy them using EKS. We will also introduce Elastic Inference (EI) accelerator which can increase the throughput while reducing the cost. EKS clusters have pods that host the inference endpoints as webservers. As the last topic for EKS-based deployment, we will introduce how the pods can be scaled horizontally for the dynamic incoming traffic.
+
+# TODO: please add notebooks
```

Comment:
@leninworld

Reply:
@ljj7975  We are not adding any EKS scripts. We only upload the SageMaker. So EKS folder is to be removed.

Reply:
@ljj7975 EKS related details (links) added